scripts/Tools/archive_metadata

```diff
@@ -1448,6 +1497,13 @@ def main_func(options):
                             case_dict['q_casename'], case_dict['query_expdb_url'])
                 sys.exit(1)
 
+    # check if expType argument is specified
+    if not options.expType:
+        logger.info('ERROR archive_metadata requires a valid expType. '\
```

Comment:
this should not be logger.info - I think that logger.ERROR is the appropriate level.

Reply:
fixed